conformance/lib/build.gradle.kts

```diff
@@ -37,23 +37,8 @@ kotlin {
         }
 
         val commonTest by getting {
-            dependencies {
-                implementation(kotlin("test-common"))
-                implementation(kotlin("test-annotations-common"))
-            }
-        }
-
-        val jvmTest by getting {
             dependencies {
                 implementation(kotlin("test"))
-                implementation(kotlin("test-junit"))
-                implementation("junit:junit:4.12")
-            }
-        }
-
-        val jsTest by getting {
-            dependencies {
-                implementation(kotlin("test-js"))
```

Comment:
Whatever this dependency provided is included in `kotlin("test")` now?

Reply:
Yep. Kotlin 1.5 simplified the test dependencies so you can now just add a `kotlin("test")` dependency to the `common` source set and it'll pull in the correct dependencies on each platform automatically.